### PR TITLE
fix GEARPUMP-117 add scripts to start and stop the cluster

### PR DESCRIPTION
**Pull request by huafengw**

### CODE REVIEW

File: bin/config.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                MASTERPORTS+=(3000)
+            else
+                MASTERPORTS+=(${PORT})
+            fi
+        fi
+    done < "$MASTERS_FILE"
+}
+
+readSlaves() {
+    SLAVES_FILE="${GEAR_CONF_DIR}/slaves"
```
Comment (whjiang) [inline bin/config.sh]: better to make naming consistent. E.g. Gearpump has "worker", but not "slave". So, it is better to rename "slaves" to "workers".

File: bin/gear-daemon.sh [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+USAGE="Usage: gear-daemon.sh (start|stop|stop-all) (local|master|worker) [args]"
+
+OPERATION=$1
+DAEMON=$2
+ARGS=("${@:3}") # get remaining arguments as array
+
+case $DAEMON in
+    (master)
+        BASH_TO_RUN=master
```
Comment (whjiang) [inline bin/gear-daemon.sh]: do we support run web services service via this script?

Comment (huafengw) [inline bin/gear-daemon.sh]: I didn't add it, do you think it's better to support?

Comment (whjiang) [inline bin/gear-daemon.sh]: yes. Better to do it as web service is also part of GP deployment.

File: docs/deployment-standalone.md [hunk 3]
```diff
@@ -55,3 +55,12 @@ The default username and password is "admin:admin", you can check
 ![Dashboard](img/dashboard.gif)
 
 **NOTE:** The UI port can be configured in `gear.conf`. Check [Configuration Guide](deployment-configuration.html) for information.
+
+### Bash tool to start cluster
+
+There is a bash tool `bin/start-cluster.sh` can launch the cluster conveniently. You need to change the file `conf/masters`, `conf/workers` and 'conf/dashboard' to specify the corresponding machines.
```
Comment (manuzhang) [inline docs/deployment-standalone.md]: 'conf/dashboard'  is not in correct format

**Comment by huafengw** [comment]
Updated

**Comment by manuzhang** [comment]
LGTM +1

**Comment by whjiang** [comment]
+1.

**Pull request closed without merging**